pkg/server/datastore/sqlstore/sqlstore_test.go

```diff
@@ -3693,6 +3693,52 @@ func (s *PluginSuite) TestDeleteBundleDissociateRegistrationEntries() {
 	s.Require().Empty(entry.FederatesWith)
 }
 
+func (s *PluginSuite) TestListRegistrationEntriesEvents() {
```

Comment:
looks like tehre is no unit tests for pruning, I think we must validate that path too

Reply:
added test for pruning